lib/modules/manager/nuget/util.ts

```diff
@@ -141,6 +141,20 @@ export async function getConfiguredRegistries(
     }
   }
 
+  // Deduplicate registries
+  const registryUrlsWithSourceMappings = registries.map(
+    (r) => r.sourceMappedPackagePatterns + r.url,
+  );
+  registries = registries.filter((r) => {
+    const lookupKey = r.sourceMappedPackagePatterns + r.url;
+    return (
+      !registryUrlsWithSourceMappings.includes(
+        lookupKey + '#protocolVersion=2',
```

Comment:
I don't know a protocol version 2 🤔 have you seen it in the hosted app?

Reply:
No, I just was trying to be safe. Drop it?

Reply:
yes, that version doesn't exist